minio/go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/gofiber/storage/minio
+
+go 1.20
+
+require (
+	github.com/gofiber/utils v1.1.0
```

Comment:
All the other drivers were migrated to `utils/v2`

Reply:
The stretchr/testify package was used for testing